### DISCUSSION TITLE
Feature: merge Elementals

### DISCUSSION THREAD

**Discussion by 27Carat** [elemgame/elemgame.app#13] [Ideas]
1+1
I think it makes sense to add a function for combining elementals of the same type, starting with legendary rarity. For example, I have two legendary rank 1 fire elementals, I combine them and get one rank 2 elemental.
XP
Or add a leveling system, farm experience and spend it on leveling up elementals. And here you can introduce another pumping system - if I pumped two rare elementals of the same type to rank ten, I can combine them and get one epic elemental of the first rank.
Experience can be farmed in events by playing the game, or received for tasks.
Pools
Legendary elementals of the tenth rank can be placed in the pool and receive tokens or rewards. The betting pool can be made in the form of a mini-game - where elementals guard a tower or crystal from enemy invasion.

**Comment by ilyar** [comment]
We are considering this feature; it will definitely happen, thanks for the creative details.

**Comment by Vasyar83** [comment]
You can also apply it to the merging of elementals to not only increase their level by merging 1+1=2, but also by merging 3+3=4, adding a level 2 armor or protection. Accordingly, merge levels 2+3 to get 4 + 1 protection, etc. It's also possible to merge two different types of elementals, with a note specifying which elemental’s level I would like to increase. Accordingly, the degree of protection should be different; it can also be strong and specifically work for an individual character.

**Comment by 27Carat** [comment]
This is a good idea. The higher the elemental level, the more characteristics it has. But mixing two elementals of different elements, I think this is not entirely correct. But creating a fifth elemental of all four elements is more interesting. Just this elemental can become the key for special tasks or quests. Or become an NFT for staking in pools. He can be classified into a special category, for example titanium or the deity of the four elements.
In order not to confuse him with ordinary elementals, you can come up with special events or battles for him, where he will compete with other elementals of the four elements. And without affecting the basic concept and mechanics of the game.
You can also add an expanded and special improvement system to it.
